enhancements/machine-api/machine-instance-lifecycle.md

```diff
@@ -135,6 +135,10 @@ For it to happen in a non disruptive manner for actuators this proposal is inten
 
 Once this is settled we might consider to simplify the interface.
 
+### Machine lifecycle diagram
+
+![image](https://user-images.githubusercontent.com/20123872/74226366-b9069380-4cbc-11ea-9e51-b1ccc2a1b35b.png)
```

Comment:
Can we link this to the checked in image? Do relative image links work? 🤔

Reply:
They should, thanks for pointing at it